public/docs/_examples/component-styles/dart/lib/hero_controls_component.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:angular2/core.dart';
+import 'hero.dart';
+
+// #docregion inlinestyles
+@Component(
+    selector: 'hero-controls',
+    template: '''
+    <style>
```

Comment:
Is there a reason this is inline style instead of 
```dart
@Component(
    selector: 'hero-controls',
    styles: const ['''
      button {
        background-color: white;
        border: 1px solid #777;
      }'''],
    template: '''
    <h3>Controls</h3>
    <button (click)="activate()">Activate</button>
  ''')
```

Are such inline styles even shimmed correctly for style encapsulation?